compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -407,11 +407,10 @@ static void PrintMethodFields(
         "  if (($method_new_field_name$ = $service_class_name$.$method_new_field_name$) == null) {\n"
         "    synchronized ($service_class_name$.class) {\n"
         "      if (($method_new_field_name$ = $service_class_name$.$method_new_field_name$) == null) {\n"
-        "        $service_class_name$.$method_new_field_name$ = $method_new_field_name$ = \n"
```

Comment:
This little change would be quite involved as all the generated code will be changed in build. I'd suggest not to change that as it does not hurt us. Actually I believe there are other places with the same tailing space. Instead of spending time finding them, maybe leave it as is would be fine.

Reply:
OK